AirCasting/SessionViews/SessionHeaderView.swift

```diff
@@ -37,7 +39,9 @@ struct SessionHeaderView: View {
                     DeleteView(viewModel: DefaultDeleteSessionViewModel(), deleteModal: $deleteModal)
                 })
             nameLabelAndExpandButton
-        }
+        }.onChange(of: isCollapsed, perform: { value in
+            isCollapsed ? (chevronIndicator = "chevron.down") :  (chevronIndicator = "chevron.up")
```

Comment:
😎 nice, I noticed that we always have the chevron down indicator and it bothered me so good job!